server/controllers/postUser.js

```diff
@@ -1,16 +1,43 @@
 const postUserQuery = require("../database/queries/addUsers");
+
+const joi = require("@hapi/joi");
+const validationObject = {
+  username: joi // username should be string and at least it contains 2 characters
+    .string()
+    .min(2)
+    .max(40),
```

Comment:
So here you are saying username can't be more than 40 characters, which is okay but its not connsisient with the DB constraint where you set the username as 50 characters: https://github.com/GSG-K3/code-academy-wiki/blob/master/server/database/config/dbBuild.sql#L67-L74